From dd47a12fd7d6681cbfd1fd90226d6902bf98b82a Mon Sep 17 00:00:00 2001
From: Sujoy Ray <sujoy.ray@intel.com>
Date: Thu, 13 Jan 2022 14:18:30 -0800
Subject: [PATCH] Change the flash part of QEMU AST2600 evb board

This patch changes the default flash part of AST2600 evb board in
QEMU build from w25q512jv to mx66l1g45g. As Intel BMC build requires
128 MByte flash, this change is necessary to boot the BMC image into
the QEMU build.

Tested
Using the following command, the OpenBMC image could be booted
successfully under QEMU environment:

qemu-system-arm -m 1G -M ast2600-evb -nographic \
-drive file=./image-mtd,format=raw,if=mtd -net nic -net user,\
hostfwd=:127.0.0.1:2222-:22,hostfwd=:127.0.0.1:2443-:443,\
hostname=qemu

Change-Id: Ic89688609dd9c433dd28bf8744a89d2217c1dad8
Signed-off-by: Sujoy Ray <sujoy.ray@intel.com>
Upstream-Status: Pending
---
 hw/arm/aspeed.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hw/arm/aspeed.c b/hw/arm/aspeed.c
index 9a15cebe2..057a6590c 100644
--- a/hw/arm/aspeed.c
+++ b/hw/arm/aspeed.c
@@ -1291,9 +1291,9 @@ static void aspeed_machine_ast2600_evb_class_init(ObjectClass *oc, void *data)
     amc->soc_name  = "ast2600-a3";
     amc->hw_strap1 = AST2600_EVB_HW_STRAP1;
     amc->hw_strap2 = AST2600_EVB_HW_STRAP2;
-    amc->fmc_model = "mx66u51235f";
-    amc->spi_model = "mx66u51235f";
-    amc->num_cs    = 1;
+    amc->fmc_model = "mx66l1g45g";
+    amc->spi_model = "mx66l1g45g";
+    amc->num_cs    = 2;
     amc->macs_mask = ASPEED_MAC0_ON | ASPEED_MAC1_ON | ASPEED_MAC2_ON |
                      ASPEED_MAC3_ON;
     amc->i2c_init  = ast2600_evb_i2c_init;
-- 
2.25.1

